modules/vectra.py

```diff
@@ -3728,3 +3723,29 @@ def delete_match_assignment(self, uuid=None, device_serial=None):
             json=payload,
             verify=self.verify,
         )
+
+
+class V2_latest(VectraClientV2_5):
```

Comment:
Similar to V3_latest, I recommend ClientV3_latest

Reply:
Updated class name to ClientV2_latest